core/src/main/java/com/linecorp/armeria/common/logging/PrometheusMetricRequestDecorator.java

```diff
@@ -0,0 +1,299 @@
+                                                        .map(MetricLabel::name)
+                                                        .toArray(String[]::new);
+
+            timer = create(namePrefix + "_timer",
+                           collectorRegistry,
+                           name -> addQuantiles(Summary.build())
+                                   .name(name)
+                                   .labelNames(metricLabelsAsString)
+                                   .help(help)
+                                   .create());
+            success = create(namePrefix + "_success",
+                             collectorRegistry,
+                             name -> Counter.build().name(name)
+                                            .labelNames(metricLabelsAsString)
+                                            .help(help)
+                                            .create());
+            failure = create(namePrefix + "_failure",
+                             collectorRegistry,
+                             name -> Counter.build().name(name)
+                                            .labelNames(metricLabelsAsString)
+                                            .help(help)
+                                            .create());
+            activeRequests = create(namePrefix + "_activeRequests",
+                                    collectorRegistry,
+                                    name -> Gauge.build().name(name)
+                                                 .labelNames(metricLabelsAsString)
+                                                 .help(help)
+                                                 .create());
+            requestBytes = create(namePrefix + "_requestBytes",
+                                  collectorRegistry,
+                                  name -> addQuantiles(Summary.build())
+                                          .name(name)
+                                          .labelNames(metricLabelsAsString)
+                                          .help(help)
+                                          .create());
+            responseBytes = create(namePrefix + "_responseBytes",
+                                   collectorRegistry,
+                                   name -> addQuantiles(Summary.build())
+                                           .name(name)
+                                           .labelNames(metricLabelsAsString)
+                                           .help(help)
+                                           .create());
+        }
+
+        private Summary.Builder addQuantiles(Summary.Builder builder) {
+            return builder.quantile(.5, .01)
+                          .quantile(.75, .01)
+                          .quantile(.95, .01)
+                          .quantile(.98, .01)
+                          .quantile(.99, .01)
+                          .quantile(.999, .01);
+        }
+
+        private <T extends Collector> T create(String name,
```

Comment:
This `create` method doesn't seem to really be a win, it's just rearranging the arguments basically.